requirements/edx/github.txt

```diff
@@ -57,6 +57,7 @@ git+https://github.com/edx/edx-lint.git@ed8c8d2a0267d4d42f43642d193e25f8bd575d9b
 git+https://github.com/edx/ecommerce-api-client.git@1.1.0#egg=ecommerce-api-client==1.1.0
 -e git+https://github.com/edx/edx-user-state-client.git@30c0ad4b9f57f8d48d6943eb585ec8a9205f4469#egg=edx-user-state-client
 -e git+https://github.com/edx/edx-proctoring.git@release-2015-07-29#egg=edx-proctoring
+-e git+https://github.com/edx/edx-organizations.git@release-2015-08-03#egg=edx-organizations
```

Comment:
head's up. I don't know the details on this, but what I've heard from DevOps, we're also supposed to be using a ...==x.y.z as well

I think those are supposed to align with the version that is registered in the setup.py file.

Tagging @fredsmith to confirm or deny.